controllers/kogitoserverlessbuild_controller_test.go

```diff
@@ -31,11 +31,12 @@ import (
 
 func TestKogitoServerlessBuildController(t *testing.T) {
 	namespace := t.Name()
-	ksw := test.GetKogitoServerlessWorkflow("../config/samples/"+test.KogitoServerlessWorkflowSampleYamlCR, namespace)
+	ksw := test.GetBaseServerlessWorkflowNoPackage(namespace)
```

Comment:
What package means?

Reply:
We have some test that needs as  a path "../config/samples/" to load the yaml's files, others "../../config/samples/" because they are inside a package

Reply:
I think we should use a smarter way of doing that. Maybe we can infer the path we are and compose it dynamically. If you think it is worth a shot, it would be a great improvement. I wouldn't do it if it takes too much time.

Reply:
Ok, since the performance aren't the priorities in the test I can use something to identify who is the caller and use a different path

Reply:
@ricardozanini  is it ok for your this https://github.com/kiegroup/kogito-serverless-operator/pull/139/files#diff-8affcd5838f425d5db07c12738684112559a18621ec539e9b33986abe804db7fR124-R132 in conjunction with the runtime.Caller ? No other ideas to decide the path, but open if you have better

Reply:
See my comment here: https://github.com/kiegroup/kogito-serverless-operator/pull/139#discussion_r1213170430

😆